WeakAurasOptions/AuthorOptions.lua

```diff
@@ -1451,8 +1451,8 @@ function addAuthorModeOption(options, args, data, order, prefix, i)
     buttonWidth = buttonWidth + 0.15
   end
   local optionClass = optionClasses[option.type]
-  local optionName = option.class == "noninteractive" and WeakAuras.author_option_types[option.type]
-                     or option.name or L["Option #%i"]:format(i)
+  local optionName = optionClass == "noninteractive" and WeakAuras.author_option_types[option.type]
```

Comment:
Since this is fixed, it should be an invariant that the option name we display is never falls back to “Option #i”. So remove that part, it concealed this bug in the first place.